test/AWSCredentials.jl

```diff
@@ -488,39 +472,34 @@ end
             "AWS_ACCESS_KEY_ID" => test_values["AccessKeyId"],
             "AWS_SECRET_ACCESS_KEY" => test_values["SecretAccessKey"]
         ) do
-            # TODO: Remove qualification when dep on AWSCore is removed
-            aws_creds = AWS.env_var_credentials()
+            aws_creds = env_var_credentials()
             @test aws_creds.access_key_id == test_values["AccessKeyId"]
             @test aws_creds.secret_key == test_values["SecretAccessKey"]
         end
     end
 
     @testset "Instance - EC2" begin
-        apply([http_get_patch]) do
-            # TODO: Remove qualification when dep on AWSCore is removed
-            result = AWS.ec2_instance_credentials()
+        apply([_http_request_patch]) do
```

Comment:
You can also just pass in the patch directly:
```suggestion
        apply(_http_request_patch) do
```